src/commands/kv/mod.rs

```diff
@@ -61,6 +63,15 @@ fn print_error(e: ApiFailure) {
     }
 }
 
+// For handling cases where the API gateway returns errors via HTTP status codes
+// (no KV error code is given).
+fn give_status_code_context(status_code: StatusCode) {
```

Comment:
this should probably eventually be pulled up and out bc the same issue would come up on any request to any api endpoint; however there's no use doing that now.